docker-compose.dev.yml

```diff
@@ -0,0 +1,22 @@
+version: '3'
```

Comment:
What's the difference between docker-compose.yml and docker-compose.dev.yml?

Reply:
When using configuration from docker-compose.dev.yml, next is started in development mode and supports hot reloading the changes through volumes. However, the docker-compose.yml file has settings tuned for deployment and serves the built and optimized server.

Reply:
ok